PhysicsTools/Heppy/python/physicsobjects/Jet.py

```diff
@@ -49,6 +49,48 @@ def _physObjInit(self):
         self._leadingTrack = None
         self._leadingTrackSearched = False
 
+    def rawEnergy(self):
+        return (self.p4()*self.rawFactor()).energy()
```

Comment:
I was lazy and copied that line from the already existing jetID() method. Now both the new rawEnergy method and the jetID one are fixed.